src/inmanta/server/services/orchestrationservice.py

```diff
@@ -665,6 +666,8 @@ async def _put_version(
         pip_config: Optional[PipConfig] = None,
         *,
         connection: asyncpg.connection.Connection,
+        module_version_info: Optional[dict[str, str]],
+        type_to_module_data: Optional[dict[ResourceIdStr, str]],
```

Comment:
This one I don't understand

Reply:
Oh, I think I was confusing resourceTypes and resourceIds.
Here I'm linking resource ids to modules, but this should be linking resource types to modules, do you agree ?